pkg/kubectl/cmd/resource/get.go

```diff
@@ -303,8 +303,10 @@ func (options *GetOptions) Run(f cmdutil.Factory, cmd *cobra.Command, args []str
 
 	showKind := options.ShowKind
 	// TODO: abstract more cleanly
-	if resource.MultipleTypesRequested(args) || cmdutil.MustPrintWithKinds(objs, infos, sorter) {
-		showKind = true
+	if !showKind {
```

Comment:
what does this change?

Reply:
What is the complexity of the checks below? Is there any IO involved, e.g. network?

Reply:
there's no IO involved, just some logic improvement. If we already set ShowKind, there's absolutely no need to check MultipleTypesRequested or MustPrintWithKinds to decide we need to show kind.

Reply:
If so, why not changing directly to
```
showKind := options.ShowKind || resource.MultipleTypesRequested(args) || cmdutil.MustPrintWithKinds(objs, infos, sorter)
```

Reply:
seems more elegant, will update soon, thanks!